doc/man/8/ceph-fuse.rst

```diff
@@ -33,6 +33,10 @@ Options
 
 Any options not recognized by ceph-fuse will be passed on to libfuse.
 
+.. option:: -o opt,[opt...]
+
+   mount options.
+
 .. option:: -d
 
    Detach from console and daemonize after startup.
```

Comment:
@batrick I should have ask this last week:

`./bin/ceph-fuse -h` displays `-d` from `fuse_usage();` as `-d   -o debug          enable debug output (implies -f)`. 

But in the doc it is different: `Detach from console and daemonize after startup.` What is your opinion about this ?

Also, I believe in `fuse_usage();` output, `-o` is wrongly displayed too.

Reply:
I think this doc is wrong. Please confirm through testing and creating a new tracker ticket.